migration/wcmp-upgrade-migration-v4-2-1.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use migration\WCMP_Upgrade_Migration;
+
+if (! defined('ABSPATH')) {
+    exit;
+}
+
+if (class_exists('WCMP_Upgrade_Migration_v4_2_1')) {
+    return new WCMP_Upgrade_Migration_v4_2_1();
+}
+
+/**
+ * Migrates pre v4.2.1 settings
+ */
+class WCMP_Upgrade_Migration_v4_2_1 extends WCMP_Upgrade_Migration
+{
+    /**
+     * @var void
+     */
+    private $defaultExportSettings = [];
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
```

Comment:
Construct bevat niks, kan dus weg

Reply:
Niet helemaal waar
`Call to protected WCMP_Upgrade_Migration::__construct() from invalid context`

Reply:
Wow, bizar.